autometa/common/external/diamond.py

```diff
@@ -347,7 +472,10 @@ def add_taxids(hits, database, verbose=True):
     hits : dict
         {qseqid: DiamondResult, ...}
     database : str
-        </path/to/prot.accession2taxid.gz>
+        Path to prot.accession2taxid.gz database
+        e.g. </path/to/prot.accession2taxid.gz>
```

Comment:
This line is not needed.

Reply:
I am not sure which line(s) are you pointing towards.
Also, I kept both the verbal description as well as the example to maintain consistency throughout the codebase.
Let me know if you only need the description, I would the remove the "e.g." from other function like `blast` as well

Reply:
Line 476. It doesn't add any information that is not in line 475, especially because you specify the exact filename in line 475.